app/scripts/controllers/metametrics.test.ts

```diff
@@ -36,12 +55,12 @@ const MOCK_TRAITS = {
   test_bool_array: [true, true, false],
   test_string_array: ['test', 'test', 'test'],
   test_boolean_array: [1, 2, 3],
-};
+} as MetaMetricsUserTraits;
```

Comment:
This is unfortunate. I thought these tests would have used real traits. Oh well. :(